sdk/mediaservices/Microsoft.Azure.Management.Media/src/Microsoft.Azure.Management.Media.csproj

```diff
@@ -6,15 +6,15 @@
     <PropertyGroup>
         <PackageId>Microsoft.Azure.Management.Media</PackageId>
         <Description>Provides developers with libraries for managing Azure Media Services using the Azure Resource Manager API.</Description>
-        <Version>2.0.5</Version>
+        <Version>3.0.0</Version>
```

Comment:
Would 2.1.0 be enough of a jump for the project?

Reply:
All the other languages SDKs are increasing the major version, so I kept the same pattern.